spec/cube_cache_control_spec.rb

```diff
@@ -86,22 +86,32 @@
       when 'mysql', 'jdbc_mysql', 'postgresql', 'oracle'
         @connection.execute 'CREATE TABLE sales_copy AS SELECT * FROM sales'
       when 'mssql', 'sqlserver'
-        @connection.execute 'SELECT * INTO sales_copy FROM sales'
+        @connection.raw_connection.execute 'SELECT * INTO sales_copy FROM sales'
```

Comment:
I looked at the source of `@connection.execute`:

```ruby
def execute(sql, name = nil, binds = nil)
  sql = suble_binds to_sql(sql, binds), binds if binds
  if name == :skip_logging
    _execute(sql, name)
  else
    log(sql, name) { _execute(sql, name) }
  end
end
```

And source of `@connection._execute` is

```ruby
def _execute(sql, name = nil)
  @connection.execute(sql)
end
```

And source of `@connection.raw_connection` is

```ruby
def raw_connection
  @connection
end
```

which seems to me that `@connection.execute` should call `@connection.raw_connection.execute` method. So I cannot understand why you are getting different results.